boards/remote-pa/include/board.h

```diff
@@ -70,11 +70,10 @@
  * 2.4GHz RF switch controlled by SW
  * @{
  */
-#define RF_SWITCH_PORT      GPIO_D
-#define RF_SWITCH_PIN       (4)
-#define RF_SWITCH_EXTERNAL  (RF_SWITCH_PORT->DATA |= (1 << RF_SWITCH_PIN))
-#define RF_SWITCH_INTERNAL  (RF_SWITCH_PORT->DATA &= ~(1 << RF_SWITCH_PIN))
-#define RF_SWITCH_TOGGLE    (RF_SWITCH_PORT->DATA ^= (1 << RF_SWITCH_PIN))
+ #define RF_SWITCH_GPIO      GPIO_PD4
+ #define RF_SWITCH_SUB_GHZ   gpio_set(RF_SWITCH_GPIO)
+ #define RF_SWITCH_2_4_GHZ   gpio_clear(RF_SWITCH_GPIO)
+ #define RF_SWITCH_TOGGLE    gpio_toggle(RF_SWITCH_GPIO)
```

Comment:
well that depends on the use-case, actually these boards have 2 radios (sub GHz and 2.4GHz). They (can) share the same antenna, hence you may want to use only one at a time which makes this short-hand macro quite useful, or not?